validation-test/compiler_crashers_2_fixed/0153-rdar36497404.swift

```diff
@@ -0,0 +1,44 @@
+// RUN: %target-build-swift -emit-module -o %t %s
```

Comment:
It should be possible to produce a smaller test case, and it should go in test/Serialization

Reply:
It should still be in test/Serialization since its testing serialization

Reply:
I've put it into crashers because it was a crash, but I can put it where ever you tell me to put it.